package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "discord-backup",
-  "version": "3.3.2",
-  "description": "A complete framework to facilitate server backup using discord.js v12",
+  "version": "3.4.0",
```

Comment:
Also please don't bump the package.json by yourself when submitting a PR, so we can merge it later, or merge several PRs and release a major version instead!